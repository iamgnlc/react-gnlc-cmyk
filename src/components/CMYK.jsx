import React, { memo } from 'react';

import 'animate.css';

import '../assets/styles/index.scss';

import Head from './Head';
import GNLC from './GNLC';
import Repo from './Repo';

const CMYK = memo(() => {
  return (
    <div className="cmyk">
      <Head />
      <GNLC />
      <Repo />
    </div>
  );
});

CMYK.displayName = 'CMYK';

export default CMYK;
