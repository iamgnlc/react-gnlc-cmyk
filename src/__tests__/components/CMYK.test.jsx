import React from 'react';
import { render } from '@testing-library/react';

import CMYK from '../../components/CMYK';

describe('CMYK', () => {
  it('should render correctly', () => {
    const view = render(<CMYK />);

    expect(view.container).toMatchSnapshot();
  });
});
