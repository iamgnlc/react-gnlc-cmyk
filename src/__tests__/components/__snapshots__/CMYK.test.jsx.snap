// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CMYK should render correctly 1`] = `
<div>
  <div
    class="cmyk"
  >
    <div
      class="animated fadeInRight letter g"
    >
      <img
        alt="g"
        class="animated fadeInLeft"
        src="g.svg"
      />
    </div>
    <div
      class="animated fadeInRight letter n"
    >
      <img
        alt="n"
        class="animated fadeInLeft"
        src="n.svg"
      />
    </div>
    <div
      class="animated fadeInRight letter l"
    >
      <img
        alt="l"
        class="animated fadeInLeft"
        src="l.svg"
      />
    </div>
    <div
      class="animated fadeInRight letter c"
    >
      <img
        alt="c"
        class="animated fadeInLeft"
        src="c.svg"
      />
    </div>
    <div
      class="repo hide"
    />
  </div>
</div>
`;
